packages/channel-client/src/types.ts

```diff
@@ -48,6 +48,13 @@ export interface ChannelClientInterface<Payload = object> {
   closeChannel: (channelId: string) => Promise<ChannelResult>;
   getAddress: () => Promise<string>;
   getEthereumSelectedAddress: () => Promise<string>;
+  approveBudgetAndFund(
+    playerAmount: string,
+    hubAmount: string,
+    playerDestinationAddress: string,
+    hubAddress: string,
+    hubDestinationAddress: string
```

Comment:
Where is the documentation on this method? I'm confused by `hubAddress` is necessary.

Reply:
This change is just mirroring the method on the `ChannelClient` class to the interface for that class. Maybe a question for @lalexgap ?